includes/classes/CatalogBuilder.php

```diff
@@ -43,10 +43,8 @@ public function catalog( $post_id, $opts = [] ) {
 
 	/**
 	 * Resets the Block Catalog by removing all catalog terms.
-	 *
-	 * @param array $opts Optional opts
 	 */
-	public function delete_index( $opts = [] ) {
+	public function delete_index() {
```

Comment:
@dsawardekar Is there any reason to remove this param? If you are refactoring, then can you check compatibility? I notice that few functions are passing  `$opt`.

<img width="786" alt="image" src="https://github.com/10up/block-catalog/assets/1784821/6d756371-cb58-4e69-b11d-c007e4ceae3f">

Reply:
@ravinderk I've added back the extra opts.